### ISSUE TITLE
docs: Confusion about importing Routed Feature Modules

### ISSUE THREAD

**Issue by shairez** [angular/angular.io#2298]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[x] Docs clarification
```
Following our conversation @wardbell -

In the [ngmodule-faq](https://angular.io/docs/ts/latest/cookbook/ngmodule-faq.html) doc, there is a line which states - 
"Routed Feature Modules are never imported."

But in the [ngmodule guide]() itself, the ContactModule looks like a Routed Feature Module.

If contains a `contact.routing.ts` file, and from the `app.routing.ts` you have a redirection to the contact path.

It can be found both in examples and in the final plunker at the end of the doc.

So, I'm trying to understand what am I missing, is there a reason the `ContactModule` is imported in the `app.module.ts` when it isn't suppose to be?

Thanks!

**Comment by wardbell** [comment]
It doesn't hurt. But it's a mistake. I'll fix.  Hang tight.

**Comment by shairez** [comment]
Losing sleep over this @wardbell 
Can't wait !!!! 

;)

**Comment by wardbell** [comment]
Looks like the mistake is in the statement. A routed component doesn't have to be exported but it must be declared in a module and, unless that module is lazy loaded, that module must be imported. Fixed by pr #2304

**Comment by shairez** [comment]
Thanks for the fix @wardbell !
